openquake/calculators/views.py

```diff
@@ -1740,7 +1740,7 @@ def view_aggrisk(token, dstore):
     AVG = len(rlzs)
     arr = numpy.zeros(AVG + 1, dt)
     for r, rlz in enumerate(rlzs):
-        arr[r]['gsim'] = repr(repr(rlz.value[0]))
```

Comment:
@micheles I hope I did not break anything by replacing the double `repr` with a single one. If that's a problem, I can modify the string afterwards while presenting the data.